core/kazoo_number_manager/src/carriers/knm_local.erl

```diff
@@ -20,56 +20,90 @@
 
 -include("knm.hrl").
 
+-define(PREFER_AVAILABLE_NUMBERS,
+        kapps_config:get_is_true(?KNM_CONFIG_CAT, <<"search_prefers_available_numbers">>, 'false')).
+
 %%--------------------------------------------------------------------
 %% @public
 %% @doc
-%% Query the local system for a quanity of available numbers
+%% Query the local system for a quantity of available numbers
 %% in a rate center
 %% @end
 %%--------------------------------------------------------------------
 -spec find_numbers(ne_binary(), pos_integer(), kz_proplist()) ->
-                          {'error', 'not_available'}.
-find_numbers(Number, Quanity, Opts) when size(Number) < 5 ->
-    find_numbers(<<"+1", Number/binary>>, Quanity, Opts);
-find_numbers(_Number, _Quanity, _Opts) ->
-    %% TODO: given the requestor's account, discover knm_local numbers
-    %%        that are available but managed by accendants of the account.
-    {'error', 'not_available'}.
+                          {'ok', knm_number:knm_numbers()} |
+                          {'error', any()}.
+find_numbers(Number, Quantity, Options) when size(Number) < 5 ->
+    find_numbers(<<"+1", Number/binary>>, Quantity, Options);
+find_numbers(Number, Quantity, Options) ->
```

Comment:
why add +1 ? what about searching other countries ? i think that it should got here already with +1 or +44 or +49 ...
depending on what crossbar receives , maybe event adding a country prefix in the api payload for search.